std_msgs/Header       header
float64               Time
utc_time	      UtcTime
uint16                Week
uint8                 GpsFix
uint8                 NumSats
swarm_control/lla     LLA
swarm_control/ned     NED
geometry_msgs/Vector3 NedVel
geometry_msgs/Vector3 NedAcc
std_msgs/String       FlightMode
swarm_control/rpy     RPY
float64               Battery
swarm_control/lla     Home

